src/Services/src/File/FileService.cs

```diff
@@ -104,6 +106,14 @@ public string PathToCfCliConfigFile
                 return Path.Combine(VsixPackageBaseDir, _defaultCfCliDir, _defaultCfCliConfigFileName);
             }
         }
+        
+        public string PathToCfDebugAdapterScript
+        {
+            get
+            {
+                return Path.Combine(VsixPackageBaseDir, _cfDebugAdapterScriptDir, _cfDebugAdapterScriptName);
+            }
+        }
```

Comment:
```suggestion
        public string PathToCfDebugAdapterScript
            => Path.Combine(VsixPackageBaseDir, _cfDebugAdapterScriptDir, _cfDebugAdapterScriptName);
```
This format for simple getters can help reduce noise... Not a required change and I did type it in the browser so there could also be a silly mistake in here

Reply:
Resolved in [ae1a1e6](https://github.com/vmware-tanzu/tanzu-toolkit-for-visual-studio/pull/360/commits/ae1a1e6e5b25611e50bdf4feb25b2bae82d776f9)